meshio/exodus/_exodus.py

```diff
@@ -141,8 +141,8 @@ def read(filename):  # noqa: C901
         # elif key == "eb_names":
         #     value.set_auto_mask(False)
         #     eb_names = [b"".join(c).decode("UTF-8") for c in value[:]]
-        elif key == "node_ns":
-            ns = value
+        elif key.startswith("node_ns"):
```

Comment:
Out of interest: Why `startswith`?

Reply:
It is because the keys for node sets are `node_ns1`, `node_ns2`, etc.

Reply:
Best add a comment about that.

Reply:
OK, the comment added.